cobbler/modules/managers/import_signatures.py

```diff
@@ -514,7 +515,8 @@ def get_proposed_name(self, dirname, kernel=None):
         # remove any architecture name related string, as real arch will be appended later
         name = name.replace("chrp", "ppc64")
         for separator in ['-', '_', '.']:
-            for arch in ["i386", "x86_64", "ia64", "ppc64le", "ppc64el", "ppc64", "ppc32", "ppc", "x86", "s390x", "s390", "386", "amd"]:
+            for arch in ["i386", "x86_64", "ia64", "ppc64le", "ppc64el", "ppc64", "ppc32", "ppc", "x86", "s390x",
+                         "s390", "386", "amd"]:
```

Comment:
seems like a valid usecase for a global?

Reply:
I will open an issue with that. I would prefer to have an Enum with a bit of logic to not type them as Strings but centrally manage them in Cobbler.